projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/src/components/CreateJob.tsx

```diff
@@ -64,35 +65,40 @@ export default class CreateJobDialog extends Component<ICreateJobDialogProps> {
    */
     private _onLocalClick() {
         return (event: React.MouseEvent) => {
-            this._handleCheckboxEvent("Local", "jobPath", "local")
+            let checkbox = document.getElementById("Local");
```

Comment:
This and _onCloudClick seem to do the same thing with different input. Might be worth exporting this logic to its own method and calling it in the callbacks.

Reply:
Primarily, I did it like that but I removed it while removing the sessionStorage. Will implement it again